src/main/java/net/neoforged/neoforge/common/data/internal/NeoForgeItemTagsProvider.java

```diff
@@ -101,7 +101,7 @@ public void addTags(HolderLookup.Provider lookupProvider) {
         copy(Tags.Blocks.FENCES_NETHER_BRICK, Tags.Items.FENCES_NETHER_BRICK);
         copy(Tags.Blocks.FENCES_WOODEN, Tags.Items.FENCES_WOODEN);
         tag(Tags.Items.FERTILIZERS).add(Items.BONE_MEAL);
-        tag(Tags.Items.FOODS_FRUIT).add(Items.APPLE, Items.GOLDEN_APPLE, Items.ENCHANTED_GOLDEN_APPLE);
+        tag(Tags.Items.FOODS_FRUIT).add(Items.APPLE, Items.GOLDEN_APPLE, Items.ENCHANTED_GOLDEN_APPLE).add(Items.MELON);
```

Comment:
This is adding the item version of the Melon block, but the Fabric PR adds the melon _slice_ item. See https://github.com/FabricMC/fabric/pull/3957/files#diff-bd8d1f7f942977cc89b7d65685dd0327a87c3656352c817981c821fd142fb269R323.